src/scripts/ci_build.py

```diff
@@ -835,6 +837,13 @@ def main(args=None):
 
             cmds.append(make_prefix + make_cmd + make_targets)
 
+            if target in ['examples']:
+                cmds.append(['gcc', '-Wall', '-Wextra', '-std=c89',
+                             '-I%s' % (os.path.join(build_dir, 'build/include/public')),
+                             os.path.join(root_dir, 'src/examples/ffi.c'),
+                             '-L%s' % (build_dir), '-lbotan-3', '-o',
+                             os.path.join(build_dir, 'build/examples/ffi')])
```

Comment:
Somewhat ugly, but it does get the job done, I guess. I fully agree that this test doesn't justify actually "supporting" C compilers in our build system.

Perhaps as a quick win in making this less brittle:

```suggestion
            if target in ['examples'] and options.cc in ['clang', 'gcc']:
                cmds.append([options.cc, '-Wall', '-Wextra', '-std=c89',
                             '-I%s' % (os.path.join(build_dir, 'build/include/public')),
                             os.path.join(root_dir, 'src/examples/ffi.c'),
                             '-L%s' % (build_dir), '-lbotan-3', '-o',
                             os.path.join(build_dir, 'build/examples/ffi')])
```